code/game/objects/structures/catwalk.dm

```diff
@@ -7,11 +7,6 @@
 	density = 0
 	anchored = 1.0
 
-	footstep_sounds = list("human" = list('sound/effects/footstep/catwalk1.ogg',
-						'sound/effects/footstep/catwalk2.ogg',
```

Comment:
footstep sounds were removed from objects everywhere and moved centrally to sound.dm